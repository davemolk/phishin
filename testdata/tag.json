{"success":true,"total_entries":1,"total_pages":1,"page":1,"data":{"id":4,"name":"Jamcharts","slug":"jamcharts","group":"Curated Selections","color":"#888888","priority":4,"description":"Phish.net Jam Charts selections (phish.net/jamcharts)","updated_at":"2019-02-10T06:58:41Z","show_ids":[],"track_ids":[6,7,10,20,24,26]}}