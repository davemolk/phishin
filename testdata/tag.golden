Name:      Description:                                           Group:
Jamcharts  Phish.net Jam Charts selections (phish.net/jamcharts)  Curated Selections

Show IDs Where Jamcharts Appears
3

Track IDs Where Jamcharts Appears
1, 2
